src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/ClaimController.java

```diff
@@ -130,4 +135,11 @@ public CaseReference preSubmit(@PathVariable("externalId") String externalId,
                                    @RequestHeader(HttpHeaders.AUTHORIZATION) String authorisation) {
         return claimService.savePrePayment(externalId, authorisation);
     }
+
+    @PutMapping("/{claimReference:" + CLAIM_CITIZEN_REFERENCE_PATTERN + "}/paid-in-full/date-paid/{moneyReceived}")
```

Comment:
please use {externalId:" + UUID_PATTERN + "} instead of claimReference to uniquely identify claim e.g `value = "/{externalId:" + UUID_PATTERN + "}/request-more-time"`